packages/webapp/src/components/Task/AbandonTask/styles.module.scss

```diff
@@ -0,0 +1,13 @@
+.radioLabel > span{
+    border-radius: 32px;
+    padding: 1px 10px;
+    border: 1px solid var(--grey400);
+    background-color: var(--grey200);
+    color: var(--grey600);
+    margin-left: 5px;
+}
+.radioLabel.active > span{
+    border: 1px solid var(--teal900);
+    background-color: var(--teal700);
```

Comment:
Do you have access to figma? https://www.figma.com/file/F3f6DpGcxAaEsxfl7OZkI2/Litefarm?node-id=30575%3A100361&t=r61M3p1gOPS6hZMy-0
The colors are darker than required. You can also check font-weight and font-size.